### PR TITLE
Delete GANA package

### PR DESCRIPTION
**Pull request by Baschdl**
Delete GANA package because all sub packages are in GANA-...

### CODE REVIEW

**Comment by Baschdl** [comment]
Wait for #27 to be merged

**Comment by Baschdl** [comment]
Maybe we have to change the package name in `.smalltalk.ston` but I don't really know

File: BaselineOfGANA.package/BaselineOfGANA.class/instance/baseline..st [hunk 1]
```diff
@@ -5,7 +5,9 @@ baseline: spec
...
-			package: 'GANA' "with: [spec requires: #('Squot')]";
+			package: 'GANA-Core' "with: [spec requires: #('Squot')]";
+			package: 'GANA-Tests'
+			package: 'GANA-UI'
 			
-			group: 'default' with: #('GANA');
-			group: 'tests' with: #('GANA')]
\ No newline at end of file
+			group: 'default' with: #('GANA-Core' 'GANA-Tests' 'GANA-UI' );
+			group: 'tests' with: #('GANA-Core' 'GANA-Tests' 'GANA-UI' )]
```
Comment (Baschdl) [inline BaselineOfGANA.package/BaselineOfGANA.class/instance/baseline..st]: @tom95 Do you know if this configuration with different subpackages each listed in the baseline is possible? Do we have to somehow list them also in the file `.smalltalk.ston`?

**Comment by Baschdl** [comment]
@tom95 Our tests with Squot work on the master branch with all versions of Squeak and not only on trunk. What's the difference to this branch?  [https://travis-ci.org/HPI-SWA-Teaching/SWT17-Project-01/builds/238427203](https://travis-ci.org/HPI-SWA-Teaching/SWT17-Project-01/builds/238427203)

**Comment by tom95** [comment]
Ah interesting. This is actually my bad then. Squot can have problems on non-trunk versions of squeak since it depends on recent changes IIRC, but I think that's actually only related to saving extension methods. So I guess best way to go here would be to revert my change dropping the older versions, but possibly dropping them again, if you start using features that require newer changes and see failures popping up. (The latter procedure has the benefit that you'll also know where/what exactly would break on an older version)

**Comment by Baschdl** [comment]
Fixes #28

**Review by feeds — APPROVED** [review]

**Pull request merged**